test/integration/backward_compatible/parallel/ListenersOnReconnectTest.js

```diff
@@ -70,13 +69,20 @@ describe('ListenersOnReconnectTest', function () {
                 }
             }
         };
-        await map.addEntryListener(listener, 'keyx', true);
+        const registrationId = await map.addEntryListener(listener, 'keyx', true);
         await Promise.all([
             turnoffMember(cluster.id, members[membersToClose[0]].uuid),
             turnoffMember(cluster.id, members[membersToClose[1]].uuid)
         ]);
 
-        await TestUtil.promiseWaitMilliseconds(8000);
+        // Assert that connections are closed and the listener is reregistered.
+        await TestUtil.assertTrueEventually(async () => {
+            const activeConnectionsCount = TestUtil.getConnections(client).length;
+            expect(activeConnectionsCount).to.be.equal(1);
+            const activeRegistrations = TestUtil.getActiveRegistrations(client, registrationId);
+            expect([...activeRegistrations.keys()].length).to.be.equal(1);
```

Comment:
Shouldn't we verify that the single connection in the active connections has a registration in the active registrations?

Reply:
yeah I will do that